docs/04_cv32a65x_design/source/subsystem.rst

```diff
@@ -26,7 +26,12 @@ At least 6 cycles are needed to execute one instruction.
 Connection with other sub-systems
```

Comment:
At line 18, missing period at the end of the sentence.
At line 21 "The processor is single issue", no more true for CV32A65X.

Reply:
Thanks for the feedback, I will fix in a next PR.